products/rhel10/profiles/anssi_bp28_intermediary.profile

```diff
@@ -0,0 +1,40 @@
+documentation_complete: true
+
+metadata:
+    SMEs:
+        - marcusburghardt
+        - yuumasato
```

Comment:
We probably want to update the SMEs to include @vojtapolasek and remove @yuumasato .

Reply:
Updated.